app/helpers/application_helper.rb

```diff
@@ -60,4 +60,8 @@ def admin_header_link
   def list_from_translation_path(translation_path)
     render 'shared/forms/list_items', translation_path: translation_path
   end
+
+  def translation_exists_for?(key)
+    !/^translation missing/.match? I18n.t(key)
```

Comment:
There is an [`I18n.exists?`](https://www.rubydoc.info/github/svenfuchs/i18n/master/I18n%2FBackend%2FBase:exists%3F) that is probably a cleaner way of doing this.

Reply:
Didn't know that - thanks.